doc/man3/SSL_get_error.pod

```diff
@@ -25,6 +25,13 @@ other OpenSSL function calls should appear in between.  The current
 thread's error queue must be empty before the TLS/SSL I/O operation is
 attempted, or SSL_get_error() will not work reliably.
 
+=head1 NOTES
+
+Some TLS implementations do not send a close_notify alert on shutdown. Before
+OpenSSL 3.0, B<SSL_ERROR_SYSCALL> with errno=0 was reported in such cases.
+Since OpenSSL 3.0 the returned error is B<SSL_ERROR_SSL> with a meaningful
+error on the error stack. See the B<SSL_ERROR_ZERO_RETURN> below.
```

Comment:
It's not only in case the peer never sends a close_notify alert, but any unexpected EOF.

I would suggest that you either remove the last sentence, or talk about peers that never send it, and then just point them to directly to the SSL_OP_IGNORE_UNEXPECTED_EOF option.

Reply:
Done.